gaurav/src/pages/recipe/SavedRecipe.tsx

```diff
@@ -0,0 +1,94 @@
+import { MapPin, Play, Tags, Utensils } from "lucide-react";
+import { useEffect, useState } from "react";
+import { useLocation, useNavigate } from "react-router-dom";
+import { CardSmallDetail } from "./components/CardSmallDetail";
+
+interface IRecipeData {
+  strMeal: string;
+  strMealThumb: string;
+  strTags: string;
+  strYoutube: string;
+  strInstructions: string;
```

Comment:
same interface is being used above, 

declare and export it in a separate interfaces.ts file inside lib